app/views/sidebar/member.js

```diff
@@ -0,0 +1,23 @@
+import veact from 'veact'
+import { state } from '../../controllers'
+import { type } from '../lib'
+import { assign } from 'lodash'
+
+const view = veact()
+
+const { div, h2, a } = view.els()
+
+view.styles({
```

Comment:
Nope, one language to rule them all 😛  (that `view.styles` does inline styling).